Automattic/jetpack
# Subscriptions: Fix Subscribed button alignment

For some themes (especially Lettre) the "✓ Subscribed" button alignment had some issues, like the button label was not centered or the button was too wide.

## Proposed changes:

It fixes the  "✓ Subscribed" alignment by centering it vertically and removing the custom width.

### "Lettre" theme

#### Before

<img width="401" alt="Screenshot 2024-04-25 at 09 06 00" src="https://github.com/Automattic/jetpack/assets/4068554/b80138bd-18b9-4969-8366-b6de850ebc2a">

#### After

<img width="401" alt="Screenshot 2024-04-25 at 09 05 37" src="https://github.com/Automattic/jetpack/assets/4068554/d2aee4b9-2cdc-4595-b8f2-525e5657c7c9">
<br><br>

There should be no difference for themes where those issues didn't occur, like "Twenty Twenty-Four".

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Subscribe to the blog using the Subscribe block
* Make sure the "✓ Subscribed" button rendered by the Subscribe block looks as expected
* Check different themes
